### PR TITLE
Reorder tags

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

**Comment by alikhajeh1** [comment]
@vdmgolub can you plz review+merge?

File: docs/infracost_cloud/tagging_policies.md [hunk 1]
```diff
@@ -7,66 +7,81 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
 You can also set a custom message to be included in pull request comments to give additional context or instructions. For example, you can describe why tagging policies are important or link to your internal wiki page.
 
 <img src={useBaseUrl("img/infracost-cloud/tagging-policies/customizations.png")} alt="You can customize the pull request message." />
 
-### 5. Pull requests to monitor
+### e. Pull requests to monitor
 
-Usually users monitor all pull requests for tagging policies. However, you can also set filters, e.g. only monitor pull requests in certain repositories so you can do gradual rollouts of your policy.
+Usually users monitor all pull requests for tagging policies. However, you can also set filters, e.g. only monitor pull requests in certain repositories so you can do gradual rollouts of your policy. Once your are done, save the tagging policy.
```
Comment (vdmgolub) [inline docs/infracost_cloud/tagging_policies.md:46]:
```suggestion
Usually users monitor all pull requests for tagging policies. However, you can also set filters, e.g. only monitor pull requests in certain repositories so you can do gradual rollouts of your policy. Once you are done, save the tagging policy.
```

File: docs/infracost_cloud/tagging_policies.md [hunk 2]
```diff
@@ -7,66 +7,81 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+This means that you do not need to wait for a pull request to test your policy. Go to one of your repos from Visibility > Repos, and click on the Re-run estimate button. Any tagging policy failures will be shown.
+
+<img src={useBaseUrl("img/infracost-cloud/tagging-policies/branch-policies.png")} alt="Infracost Cloud shows you any tagging policy failures that are currently happening on your main or master branch too." />
+
+## 3. Analytics on policy coverage
+
+From the Governance > Tagging policy page, you can see the percentage of resources that are passing your tagging policies over the last 6 months. Whilst cloud vendor tools such as AWS Cost Explorer show the percentage of untagged costs, Infracost Cloud shows the percentage of infrastructure-as-code resources that are strictly following your tagging policies, which is much clearer for engineers to action and improve. This is an important KPI that FinOps teams track and improve over time to reduce the percentage of costs that cannot be categorized and allocated.
```
Comment (vdmgolub) [inline docs/infracost_cloud/tagging_policies.md:60]:
```suggestion
From the Governance > Tagging policies page, you can see the percentage of resources that are passing your tagging policies over the last 6 months. Whilst cloud vendor tools such as AWS Cost Explorer show the percentage of untagged costs, Infracost Cloud shows the percentage of infrastructure-as-code resources that are strictly following your tagging policies, which is much clearer for engineers to action and improve. This is an important KPI that FinOps teams track and improve over time to reduce the percentage of costs that cannot be categorized and allocated.
```

File: docs/infracost_cloud/tagging_policies.md [hunk 3]
```diff
@@ -7,66 +7,81 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
 :::note
 If you do not use source control integrations ([GitHub App](/docs/integrations/github_app/) or [GitLab App](/docs/integrations/gitlab_app/)), you must [implement some extra steps](/docs/guides/source_control_benefits/) for CI/CD integrations to work with tagging policies. Tagging policies do not currently work with the [Terraform Plan JSON](/docs/features/cli_commands/#option-2-terraform-plan-json) method; [contact us](mailto:hello@infracost.io) for assistance.
 :::
 
-<img src={useBaseUrl("img/infracost-cloud/tagging-policies/pull-request-tags.png")} alt="Create a pull request to test your tagging policy." />
+## How tagging policies work
 
-### 7. Analytics on policy coverage
+Tagging policies check all AWS, Azure and Google Terraform resources that support tagging, including resources that Infracost does not show cost estimates for yet. The following list describes are things that are checked by tagging policies:
```
Comment (vdmgolub) [inline docs/infracost_cloud/tagging_policies.md:78]:
```suggestion
Tagging policies check all AWS, Azure and Google Terraform resources that support tagging, including resources that Infracost does not show cost estimates for yet. The following list describes things that are checked by tagging policies:
```

**Review by vdmgolub — APPROVED** [review]

**Pull request merged**